### ISSUE TITLE
Error in check_availability() Not Found (HTTP 404). Failed to process request.

### ISSUE THREAD

**Issue by Muesgen** [16EAGLE/getSpatialData#79]
Hey @16EAGLE ,

This issue is related to #76 
I was able to solve the error

when i execute `check_availabilty()` i get the following error:

```
records <- check_availability(records)
Checking instant availability for Sentinel records...
Error in .get(paste0(cred[3], "/odata/v1/Products('", x, "')/", field),  : Not Found (HTTP 404). Failed to process request.
```

The Error occurs in the check_availability() function at line 19:24

      records[records$product_group` == "Sentinel" & 
                !records$is_gnss, ]$download_available <- .apply(records.sentinel, 
                                                                 MARGIN = 1, function(x, names = colnames(records.sentinel)) {
                                                                   as.logical(toupper(unlist(.get_odata(x$entity_id, 
                                                                                                        x$cred, field = "Online/$value"))))
                                                                 })
                                                                 
Here the Error comes from the `.get_odata()` function:
   ```
.get_odata <- function(uuid, cred, field = ""){
  lapply(uuid, function(x) content(.get(paste0(cred[3], "/odata/v1/Products('", x, "')/", field),  cred[1], cred[2])))
}
```

The problem here is that a wrong url is created. one "/" too many is created:
your current code creates:

> https://scihub.copernicus.eu/dhus//odata/v1/Products('2b17b57d-fff4-4645-b539-91f305c27c69')/Online/$value

How the URL should look like:

> https://scihub.copernicus.eu/dhus/odata/v1/Products('2b17b57d-fff4-4645-b539-91f305c27c69')/Online/$value

As you can see, there is one too many "/"s before "odata".

So when you remove the "/" in your .get_odata() function, for me as a windows user, the function .get_odata() and check_availability() works fine.

Here is the .get_odata() function  with which it works:

```
.get_odata <- function(uuid, cred, field = ""){
  lapply(uuid, function(x) content(.get(paste0(cred[3], "odata/v1/Products('", x, "')/", field),  cred[1], cred[2])))
}
```
It would be cool if you could fix this quickly.
The package is quite good by the way :)

**Comment by manan-gup** [comment]
Hey, is this R package not supported anymore? There has been no commit to it in 4 and a half months. Asking to figure out if I should depend on this package for my work or figure out another solution.

**Comment by labguyvancouver** [comment]
Has this issue been resolved in the code yet? I'm considering using the package for a project, but am getting the errors listed above. I just installed the package a few days ago, how can I go about fixing the issue?

**Comment by Muesgen** [comment]
My workaround is to fork the package. Manually change the above mentioned solution in the script and then install it.

**Comment by 16EAGLE** [comment]
Thank you for raising this issue and your patience. Should be fixed with #86. Please reopen in case you encounter the same issue again.
